Author: Apollon Oikonomopoulos <apoikos@debian.org>
Description: Start after rsyslog.service
 As HAProxy is running chrooted by default, we rely on an additional syslog
 socket created by rsyslog inside the chroot for logging. As this socket cannot
 trigger syslog activation, we explicitly order HAProxy after rsyslog.service.
 Note that we are not using syslog.service here, since the additional socket is
 rsyslog-specific.
Forwarded: no
Last-Update: 2017-12-01
--- a/contrib/systemd/haproxy.service.in
+++ b/contrib/systemd/haproxy.service.in
@@ -1,6 +1,6 @@
 [Unit]
 Description=HAProxy Load Balancer
-After=network.target
+After=network.target rsyslog.service
 
 [Service]
 Environment="CONFIG=/etc/haproxy/haproxy.cfg" "PIDFILE=/run/haproxy.pid"
